src/contentScript.js

```diff
@@ -33,51 +33,100 @@ async function freezeThisPage(options) {
  * Top-level dispatcher for commands sent from the devpanel or corpus collector
  * to this content script
  */
-async function dispatch(request) {
+function dispatch(request) {
     switch (request.type) {
+        case 'init':
+            injectSimmer();
+            break;
+
         case 'label':
-            // TODO: Show a message or something if no element is inspected.
-            elementAtPath(request.inspectedElement, document).setAttribute('data-fathom', request.label);
+            setLabel(request);
             break;
+
         case 'freeze':
-            // Freeze a page at the bidding of the corpus collector or devtools
-            // save button. Devtools panel calls this indirectly, by way of the
-            // background script, so it can download the result when done.
```

Comment:
~~I think it's useful to at least preserve this knowledge of the reason why we do the downloading in the bg script. Saying "Only background scripts are allowed to download things." in the bg script would suffice. That stuff is woefully underdocumented, and I had to go through a lot of trial and error to figure it out.~~

Never mind; I just discovered where you moved it. :-)